### ISSUE TITLE
 liqo-network-manager crash due to blank CIDR for liqoctl v0.11.0-rc.1

### ISSUE THREAD

**Issue by panpan0000** [liqotech/liqo#2656]
#### What happened:
liqoctl install will fail due to  liqo-network-manager crash , with blank CIDR 

#### How to reproduce it (as minimally and precisely as possible):

```
liqoctl install kubeadm --only-output-values
```

```
networkManager:
    config:
        additionalPools: []
        podCIDR: 
        reservedSubnets: []
        serviceCIDR: 
```

```
kubectl -n liqo get deploy liqo-network-manager -o yaml will show below
```
```
    spec:
      containers:
      - args:
        - --run-as=liqo-network-manager
        - --manager.pod-cidr=
        - --manager.service-cidr=
```
#### Environment:
liqoctl version
Client version: v0.11.0-rc.1
Server version: v0.10.3

**Comment by panpan0000** [comment]
more over, after this problem being fixed 
 liqoctl status still shows:
```
|  ERRO  unable to collect status for deployment liqo-ipam in namespace liqo:      |
|        deployments.apps "liqo-ipam" not found                                    |
|  ERRO  unable to collect status for daemonSet liqo-fabric in namespace liqo:     |
|        daemonsets.apps "liqo-fabric" not found
```

**Comment by fra98** [comment]
Hi @panpan0000, as shown on the output of `liqoctl version`, the client and server have different versions.
In your case, you need to explicitly set the right version when installing liqo (i.e., `liqoctl install kubeadm --version v0.11.0-rc.1`)

**Comment by panpan0000** [comment]
it's weird, I thought when we have the v0.11.0 liqoctl CLI, by default it will install the helm chart release as the same version (say , v0.11.0 )?

**Comment by fra98** [comment]
This behavior is fixed in PR #2655, which will be merged for the upcoming v1.0.0-rc.1 release
